src/app/business/block/cloud-file-share/create/cloud-file-share-create.component.html

```diff
@@ -10,21 +10,27 @@ <h1>{{i18n.keyID['sds_creatFileShare']}}</h1>
     <form-item label="{{label.backend}}" [required]="true">
         <p-dropdown [style]="{'min-width':'150px','width':'220px'}" placeholder="Please select" [options]="backendsOption" (onChange)="updateFormAndRegion()" formControlName="backend"></p-dropdown>
     </form-item>
+    <form-item *ngIf="cloudFileShareCreateForm.controls['sfsType']" label="{{label.sfs_type}}" [required]="true">
+        <span *ngIf = "false"><p-radioButton label="Auto" formControlName="sfsType" name="typeRadioGroup" value="Auto"></p-radioButton></span>
```

Comment:
Is this field always hidden?
`*ngIf = "false"` will set it to false always.

Reply:
always hidden means?

Reply:
```
<span *ngIf = "false"><p-radioButton label="Auto" formControlName....
```
here ngIf is set to false. There is no check on any variable. So not sure what the check is there for.
Where is the `Auto` value used?
In the screenshot above I see only `SFS` and `SFS_Turbo` as the options.

Reply:
Thanks, Done